docs/advanced-usage/connecting-cloud-storage/amazon-s3.md

```diff
@@ -0,0 +1,111 @@
+Integrations can be established to Amazon S3.
+
+To get started, ensure you have admin access within Kolena.
+Navigate to your [Organization Settings's Integrations Tab](https://app.kolena.io/redirect/organization?tab=integrations) and click "Add Integration", then "AWS".
```

Comment:
> Organization Settings's

The phrasing here is awkward (and I've noticed that this copy appears in all cloud storage integration guides). I'd say:

> Navigate to the "Integrations" tab on the Organization Settings page

And link "Organization Settings" with the proper icon, like this example from the FAQ:

<img width="173" alt="Screen Shot 2023-08-22 at 9 46 41 AM" src="https://github.com/kolenaIO/kolena/assets/9648886/215f498a-d29d-4a71-be01-b67bd227c1a2">

Reply:
To be clear I would still link to the integrations tab, just phrase it differently and only link from the "Organization Settings" text. Currently we're very careful to be consistent with links to the Kolena app, including the icon and the proper page title